src/software/convert/main_convertSfMFormat.cpp

```diff
@@ -61,7 +63,7 @@ int main(int argc, char **argv)
     ("describerTypes,d", po::value<std::string>(&describerTypesName)->default_value(describerTypesName),
       feature::EImageDescriberType_informations().c_str())
     ("imageWhiteList", po::value<std::vector<std::string>>(&imageWhiteList)->multitoken()->default_value(imageWhiteList),
-      "image white list (uids or image paths).")
+      "image white list (uid, image filename or regex on the image file path).")
```

Comment:
please give an example of the supported regex(es). 
If I understand `#` matches a single digit, `@` one or more digits, `?` one character and `*` zero or more characters? Is there a name for this simplified version of regex?
You can add `image white list containing uid(s), image filenames or regex(es) on the image file path (supported regex: # matches a single digit, @ one or more digits, ? one character and * zero or more)`